components/clearbutton/stories/clearbutton.stories.js

```diff
@@ -61,3 +61,22 @@ export const OverBackground = Template.bind({});
 OverBackground.args = {
 	staticColor: "white",
 };
+
+/**
+ * Stories for the MDX "Docs" only.
+ * Based off of the base `Template` which does not have conditional Chromatic-only markup.
+ */
+export const Disabled = Template.bind({});
+Disabled.args = {
+	isDisabled: true,
+};
+Disabled.tags = ["docs-only"];
+Disabled.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+export const Quiet = Template.bind({});
```

Comment:
Out of curiosity- is this just the default/standard variant now? I see it should have the `--quiet` modifier class, but I can't quite gather what "quiet" is supposed to do or mean.

Reply:
The quiet class was a little puzzling...I dug into this a bit and finally understand why. It's because the Express version has a background color (turn on Express in global controls + check out the themes/express.css).

Storybook for this component is missing the quiet class as a control; could you add it and make sure it's enabled for this Story? It might be worth adding a sentence for this in the MDX too, something like "The `.spectrum-ClearButton--quiet` class will use a transparent background (including when the Express theme is active)."

Reply:
Thanks for figuring that out for me! I hope it didn't take too much digging 😬

Hopefully I added that control correctly, along with that explanation suggestion. https://github.com/adobe/spectrum-css/pull/2793/commits/df64d16dfe683952d814d256ad2db3d5445d5e3c